api/server/router/swarm/cluster_routes.go

```diff
@@ -107,7 +107,12 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 		return err
 	}
 
-	id, err := sr.backend.CreateService(service)
+	encodedAuth := ""
+	if auth, ok := r.Header["X-Registry-Auth"]; ok {
```

Comment:
Still not sure about this; if another (non-docker CLI) client is sending these headers, which doesn't use this casing

should we add a to-do here for checking case-insensitive?